interface/reports/appointments_report.php

```diff
@@ -315,10 +315,10 @@ class='datepicker form-control'
                                 <a href='#' class='btn btn-default btn-print' id='printbutton'>
                                     <?php echo xlt('Print'); ?>
                                 </a>
-                                <a href='#' class='btn btn-default btn-transmit' onclick='window.open("../patient_file/printed_fee_sheet.php?fill=2","_blank")' onsubmit='return top.restoreSession()'>
+                                <a href='#' class='btn btn-default btn-transmit' onclick='window.open("../patient_file/printed_fee_sheet.php?fill=2", "_blank")' onsubmit='return top.restoreSession()'>
```

Comment:
For the follow up commit recommend trying to figure out out to incorporate the rel="noopener" into this window.open mechanism. (if need hints, just let us know :) )

Reply:
I'll be taking this on today!  Thanks!

Reply:
(has since been handled!)